### ISSUE TITLE
WSL2 : Can't deploy smart contract file.

### ISSUE THREAD

**Issue by scalalang2** [iboss-ptk/terrain#20]
When I run this command as below. It gives me an error. `account sequence mismatch`

```
$ npx terrain deploy counter --signer custom_tester_1 --network testnet
CLIError: account sequence mismatch, expected 6, got 5: incorrect account sequence
    at Object.error (/usr/lib/node_modules/@iboss/terrain/node_modules/@oclif/errors/lib/index.js:26:15)
    at Object.exports.instantiate (/usr/lib/node_modules/@iboss/terrain/lib/lib/deployment.js:68:26)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {
  oclif: { exit: 2 },
  code: undefined
}
```

**Comment by iboss-ptk** [comment]
@scalalang2  I guess this has something to do with nonce, haven't found the exact cause yet but running it few times fix the issue.

**Comment by scalalang2** [comment]
I have tried several things.
This code as below helped me to deploy contract well.

I gave a slight time term between the StoreCodeTx and the Instantiation Tx.

```typescript
const storeCode = new MsgStoreCode(
    wallet.key.accAddress,
    fs.readFileSync(`../artifacts/${wasmFilePath}`).toString('base64'),
);

const storeCodeTx = await wallet.createAndSignTx({
    msgs: [storeCode],
});

const storeCodeTxResult = await this.client.tx.broadcast(storeCodeTx);
const codeId = storeCodeTxResult.logs[0].events[1].attributes[1].value;
console.log('CodeId: ', codeId);

await new Promise((reoslve, reject) => {
    console.log("waiting to be stable.")
    setTimeout(() => {
        reoslve({})
    }, 2000)
})

const instantiate = new MsgInstantiateContract(
    wallet.key.accAddress, // owner
    undefined, // admin address
    34436, // code ID
    { ...initMsg }, // InitMsg
    {}, // init coins
);
```

**Comment by iboss-ptk** [comment]
Interesting, would find time to look deeper into the issue, this happens on the frontend from time to time for me as well.

**Comment by iboss-ptk** [comment]
This should be fixed by #25, please reopen the issue if there is still a problem
